docs/sources/alerting/fundamentals/notification-policies/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Notification Policies
+description: Introduction to Notification Policies and how they work
+weight: 409
+keywords:
+  - grafana
+  - alerting
+  - notification policies
+---
+
+# How Notification Policies work
+
+If you’ve ever used Prometheus’ Alertmanager you might be familiar with its routes and routing tree. It’s a very flexible way to route alerts to various different receivers and, with its label matchers, is a powerful abstraction that allows modifications to alert notification delivery without having to update every individual alert rule.
+
+In Grafana Alerting we refer to these routes as **Notification Policies**.
```

Comment:
```suggestion
Notification policies provide you with a flexible way of routing alerts to various different receivers. Using label matchers, you can modify alert notification delivery without having to update every individual alert rule.

Learn more about how notification policies work and are structured, so that you can make the most out of setting up your notification policies. 
```